2select-html/styles.css

```diff
@@ -2,15 +2,47 @@ body {
     font-family: sans-serif;
```

Comment:
Extra spaces here, for example

Reply:
Thanks, I am checking this out and will fix. I appreciate the suggestion to check my tab size in the editor!